client/payment-details/timeline/map-events.js

```diff
@@ -96,7 +97,7 @@ const getDepositTimelineItem = (
 
 		headline = createInterpolateElement( headline, {
 			// eslint-disable-next-line jsx-a11y/anchor-has-content
-			a: <a href={ depositUrl } />,
+			a: <Link href={ depositUrl } />,
```

Comment:
This and the below change are causing a [failing test](https://github.com/Automattic/woocommerce-payments/pull/2290/checks?check_run_id=2879438553#step:5:1576): between the two changes, it's outputting `Warning: React.createElement: type is invalid` 3 times.

Any ideas why the warning / test failure?

Reply:
@dechov it's happening because the module it's being mocked at the top. Adding `Link` to the "exported" object makes it work. On the other hand I believe this is not ideal and the real solution would be as the TODO mentions: to remove the mock and update snapshots instead.


```js
// Mock the Timeline component because it's not published at the time of writing this test.
// TODO: Remove this mock and update snapshots once the Timeline component is live.
jest.mock( '@woocommerce/components', () => {
	const Card = require.requireActual( '@woocommerce/components' ).Card;
	const Link = require.requireActual( '@woocommerce/components' ).Link;

	return {
		Card,
		Link,
		Timeline: jest.fn( () => <div>Mock timeline</div> ),
	};
} );
```

Reply:
Oh, I didn't notice that – thanks! Great to know the solution was _removing_ code 😄 (though it adds well over 1000 lines to the snapshot)